### PR TITLE
Update Version for release of skopeo.

### PR DESCRIPTION
**Pull request by rhatdan**

### CODE REVIEW

**Comment by rhatdan** [comment]
@vrothberg @mtrmac PTAL

**Comment by vrothberg** [comment]
I want https://github.com/containers/image/pull/630 and https://github.com/containers/image/pull/629 in this release. Let's wait for them to be merged.

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests.

**Comment by rhatdan** [comment]
Ok, I also found a bad crash after this. So I think we should wait.

File: version/version.go [hunk 1]
```diff
@@ -6,7 +6,7 @@ const (
 	// VersionMajor is for an API incompatible changes
 	VersionMajor = 1
 	// VersionMinor is for functionality in a backwards-compatible manner
-	VersionMinor = 7
+	VersionMinor = 8
```
Comment (mtrmac) [inline version/version.go]: BTW if we are at all serious about semantic versioning, this needs to be 2.0.0.

Comment (rhatdan) [reply version/version.go]: Did API change?

Comment (mtrmac) [reply version/version.go]: Yes.

Comment (rhatdan) [reply version/version.go]: Well lets move to 2.0.0 when we have mirroring.

**Pull request closed without merging**